edsdme/components/PartnerCardsStyles.js

```diff
@@ -840,6 +840,24 @@ export const partnerCardsStyles = css`
   .all-filters-wrapper-mobile .all-filters-footer-clear-btn-mobile:focus {
     border-color: ${blueColor};
   }
+  
+  .filter-info {
+    display: flex;
+    margin: 0 5px;
+  }
+  
+  .info-icon {
```

Comment:
@SonjaPopovic - I guess there's no icon available in spectrum that we can use out of the box for this?